server/chat.ts

```diff
@@ -971,44 +965,39 @@ export class CommandContext extends MessageContext {
 
 		if (!user.named) {
 			connection.popup(this.tr(`You must choose a name before you can talk.`));
-			return null;
+			throw new Chat.ErrorMessage(this.tr(`You must choose a name before you can talk.`));
 		}
 		if (!user.can('bypassall')) {
 			const lockType = (user.namelocked ? this.tr(`namelocked`) : user.locked ? this.tr(`locked`) : ``);
 			const lockExpiration = Punishments.checkLockExpiration(user.namelocked || user.locked);
 			if (room) {
 				if (lockType && !room.settings.isHelp) {
-					this.errorReply(this.tr `You are ${lockType} and can't talk in chat. ${lockExpiration}`);
 					this.sendReply(`|html|<a href="view-help-request--appeal" class="button">${this.tr("Get help with this")}</a>`);
-					return null;
+					throw new Chat.ErrorMessage(this.tr `You are ${lockType} and can't talk in chat. ${lockExpiration}`);
 				}
 				if (room.isMuted(user)) {
-					this.errorReply(this.tr(`You are muted and cannot talk in this room.`));
-					return null;
+					throw new Chat.ErrorMessage(this.tr(`You are muted and cannot talk in this room.`));
 				}
 				if (room.settings.modchat && !room.auth.atLeast(user, room.settings.modchat)) {
 					if (room.settings.modchat === 'autoconfirmed') {
-						this.errorReply(
+						throw new Chat.ErrorMessage(
 							this.tr(
 								`Because moderated chat is set, your account must be at least one week old and you must have won at least one ladder game to speak in this room.`
 							)
```

Comment:
```suggestion
							this.tr`Because moderated chat is set, your account must be at least one week old and you must have won at least one ladder game to speak in this room.`
```

This doesn't need to be multiline, right?